### PR TITLE
new outline for modeling risk factors

### PR DESCRIPTION
**Pull request by NathanielBlairStahn**
I created a new file since the outline we discussed today seemed different from the existing outlines for "Modeling Risk Exposure" and "Modeling Risk Effects."

Once this gets merged into master, we can each make a new branch and work separately on the individual sections within this file, and hopefully we can avoid merge conflicts.

### CODE REVIEW

**Comment by Ninicorn** [comment]
looks good!

File: docs/source/model_design/risk_factors.rst [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+What is a risk effect?
+++++++++++++++++++++++
+
+Definitions
+-----------
+
+TMREL/TMRED
++++++++++++
+
+Relative Risk
```
Comment (Ninicorn) [inline docs/source/model_design/risk_factors.rst]:
```suggestion
Relative Risk (RR)
```

File: docs/source/model_design/risk_factors.rst [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
++++++++++++++
```
Comment (Ninicorn) [inline docs/source/model_design/risk_factors.rst]:
```suggestion
++++++++++++++++
```

**Comment by NathanielBlairStahn** [comment]
> This looks good! I think we should also add in an intro (what's subjects are covered here) and learning objectives (what should you understand by the end of reading this)---those were really helpful to me in the Modeling Causes page.
> 
> We also have, in the Modeling Causes page, a 'why do we want a document that describes each {cause model} and {what dos a model document look like}, but perhaps we can reorganize where that info is after we've written up risk factors.

Good points, @beatrixkh .  I added a Todo to add an introduction and to add the sections you're referring to.  It looks like the "learning objectives" and "why do we want a document describing each model?" sections in the Modeling Causes page don't actually have any content specific to causes, so perhaps they should go in a separate document that applies to both causes and risks?  Or do you think each document should have its own separate learning objectives that are going to be different for different types of models?

**Pull request merged**